src/Executor/Informer.php

```diff
@@ -51,7 +57,12 @@ public function endTask()
         if ($this->output->getVerbosity() == OutputInterface::VERBOSITY_NORMAL && !$this->output->getWasWritten()) {
             $this->output->write("\033[k\033[1A\r<info>✔</info>\n");
         } else {
-            $this->output->writeln("<info>✔</info> Ok");
+            $endTime = round(microtime(true) * 1000);
+            $millis = $endTime - $this->startTime;
+            $seconds = floor($millis / 1000);
+            $millis = $millis - $seconds * 1000;
+            $taskTime = ($seconds > 0 ? "{$seconds}s " : "")."{$millis}ms";
+            $this->output->writeln("<info>✔</info> Ok after {$taskTime}");
```

Comment:
Unfortunately there is no gray color for output. So, I've added just square brackets.